### ISSUE TITLE
[Bug] Network problem or format not supported.(Proxy is open)

### ISSUE THREAD

**Issue by Qaz-6** [iv-org/invidious#4303]
**Describe the bug**
Activate the proxy, open the video and read the title.Why is there an error if using Innertube?
A video opens on YouTube:https://yewtu.be/watch?v=6pqbFiHHrbk
**Steps to Reproduce**
1. Go to '.settings..'
2. Click on '..proxy videos..'
3. Play the video.
4. See bug
5. Tried all servers
6. Run DASH and automatic quality
-->

**Logs**
No

**Screenshots**

https://github.com/iv-org/invidious/assets/140435478/d39da455-d8af-4a6f-96d9-119f8024eef8

**Additional context**
 - Chrome Browser (if applicable):
 - Android 12 (if applicable):
-->

**Comment by unixfox** [comment]
works for me with proxy enabled. did you really try all the instances from https://instances.invidious.io?

**Comment by unixfox** [comment]
Does it also happen in firefox without any extensions installed?

**Comment by Qaz-6** [comment]
No!
1.The application does not automatically open videos in FireFox.(in all videos)
2.Buffering is delayed during backward and foward.(But in this video)
3.Although the network quality is good, the automatic quality does not work.(All videos)
4.Only this video is displayed in 144p and the reload icon appears frequently.

**Comment by Qaz-6** [comment]
Can proxy open all videos even without ınstances?

**Comment by absidue** [comment]
@Qaz-6 I saw you commenting on the yt-dlp IP block issue earlier. Please do not abuse Invidious instances to mass-download videos.

**Comment by Qaz-6** [comment]
I don't download from invidious.

**Comment by mrgohin** [comment]
I had this issue as well.

I checked via F12 (developer console, network tab) if something weird is happening and yes it did. My "media" where invalid to my client. When I opened the media link directly the video played no problem.

Since my invidious runs behind caddy I was forced to not only set domain: localhost.localdomain but also external_port: 443. Since then everything worked on maximum setting.
